src/components/Skeleton/Skeleton.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react/types-6-0';
+
+import Skeleton from './Skeleton';
+import { Inline, Padbox, Stack, Surface } from '../layout';
+
+export default {
+  title: 'components/Skeleton',
+  component: Skeleton,
+} as Meta;
+
+export const Playground: Story<any> = (args) => <Skeleton {...args} />;
+Playground.parameters = {
+  chromatic: { disable: true },
+  screenshot: { skip: true },
+};
+
+export const Default: Story = () => {
+  return (
+    <Stack gap="lg">
+      <Surface radius="md" hasBorder>
+        <Padbox paddingSize="md">
+          <Inline align="center" gap="md">
+            <Skeleton height={100} variant="circular" width={100} />
+            <Stack gap="sm" style={{ width: 100, flex: 1 }}>
+              <Skeleton variant="text" />
+              <Skeleton variant="text" />
+              <Skeleton variant="text" />
+            </Stack>
+          </Inline>
+        </Padbox>
+      </Surface>
+      <Surface radius="md" hasBorder>
+        <Padbox paddingSize="md">
+          <Stack gap="sm">
+            <Skeleton variant="text" />
```

Comment:
The suggestions don't work for the component. It suggests types from styled-components but not the component's own props.
![Screenshot 2024-02-05 at 15 15 57](https://github.com/securityscorecard/design-system/assets/18654425/55637578-86a5-4fb0-8adc-bc98e73dc079)